### ISSUE TITLE
After 6.30, some tabs suspend and cannot be unsuspended by clicking 

### ISSUE THREAD

**Issue by nompen** [deanoemcke/thegreatsuspender#530]
Please reference the png files "Healthy" and "Unhealthy." Since the upgrade, some of my tabs suspend with no title in the "reload" screen. Clicking to reload these does not work. I have to manually change the URL by deleting all the suspend information up to the http/https director. But if there is a title in the reload screen, the click to unsuspend works just as it always has.

After the upgrade, I manually edited the suspended tab URLs in my two major working windows since NO suspended screens could be unsuspended by clicking. That was approximately 120-150 suspended tabs. Since then, some tabs suspend correctly and can be opened by clicking, but the majority suspended again and cannot be unsuspended by clicking, only by editing the URL to the correct site.

Just in case it is helpful, here are the URLs for a sample healthy and unhealthy tab. I cannot see any difference.

**Unhealthy** - chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/suspended.html#ttl=reporting%20services%20-%20SSRS%20Report%20Content%20Blank%20-%20Stack%20Overflow&uri=https://stackoverflow.com/questions/16239096/ssrs-report-content-blank

**Healthy** - chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/suspended.html#ttl=ROUND%20(Transact-SQL)%20%7C%20Microsoft%20Docs&uri=https://docs.microsoft.com/en-us/sql/t-sql/functions/round-transact-sql

**Unhealthy**
![unhealthysuspension](https://user-images.githubusercontent.com/9200840/26980599-3fd1f85e-4cf8-11e7-9266-8fa329df83a0.png)

**Healthy**
![healthysuspension](https://user-images.githubusercontent.com/9200840/26980613-4843aa1e-4cf8-11e7-9e52-20107bed7167.png)

**Comment by deanoemcke** [comment]
Both those urls seem to work for me. 
![image](https://user-images.githubusercontent.com/1051763/27000351-91e21734-4d7d-11e7-979d-91da674d12ab.png)

Is it possible that it would be fixed with a chrome restart?
Actually, before you do that, it would be really useful to see what is in the developer console of the suspended tab that does not have a title..

**Comment by nompen** [comment]
Below are developer consoles.   The blank console is a "healthy" suspended tab.  The other two are from "unhealthy" suspensions.  I included two in case there is a subtle difference I cannot see.  I spot-checked a dozen and all fit the pattern.

**Additional Info**
Chrome Version 58.0.3029.110 (64-bit)
Ubuntu VERSION="17.04 (Zesty Zapus)"

**Unhealthy**
![phpnet](https://user-images.githubusercontent.com/9200840/27002996-35b729ea-4db4-11e7-853c-1f95ddc9777f.png)


![css-store](https://user-images.githubusercontent.com/9200840/27002999-457d98b4-4db4-11e7-8c9b-7a7deca1597c.png)

**Healthy**
![tls](https://user-images.githubusercontent.com/9200840/27003023-cb5ba0de-4db4-11e7-8956-d32bd811e0e9.png)

**Comment by deanoemcke** [comment]
@nompen you closed this issue? did it resolve itself? i was going to suggest doing a ctrl + f5 (hard refresh) on the pages that aren't working. as from the screenshots, i can't see why these pages would fails but others wouldn't.

**Comment by nompen** [comment]
I did not mean to close it if I did.  I added a message with the developer console info.  If you do not see it, let me know and I will open another or re-open the other ticket if that is possible.  

I cannot get to GitHub right now to check on the ticket. 

Ctrl F5 does not refresh the problem tabs.  It works as expected with the normal ones.
 

Doyle E. (Phil) Van Zandt Creative Support Systems, LLC 225-324-7045 Mobile 225-275-7709 Preferred 225-366-7045 dvanzandt@csssupports.com     From: Dean Oemcke <notifications@github.com>
Sent: 06/10/2017 10:06 PM
To: deanoemcke/thegreatsuspender <thegreatsuspender@noreply.github.com>
Cc: nompen <user1@csssupports.com>, Mention <mention@noreply.github.com>
Subject: Re: [deanoemcke/thegreatsuspender] After 6.30, some tabs suspend and cannot be unsuspended by clicking (#530)



@nompen you closed this issue? did it resolve itself? i was going to suggest doing a ctrl + f5 (hard refresh) on the pages that aren't working. as from the screenshots, i can't see why these pages would fails but others wouldn't.

**Comment by deanoemcke** [comment]
@nompen i'm really sorry but i can't see how it's possible that the `suspended.js` script is failing on one tab, and not on another one.

the line in question is: `chrome.extension.getBackgroundPage().gsUtils`
which means on the tabs that are failing, the call to `chrome.extension.getBackgroundPage()` is not working properly.

have you done a browser or computer restart since the upgrade.
it's possible that the issue has something to do with ubuntu, that's the only thing i'm unable to test from my end.

**Comment by nompen** [comment]
I had postponed a reboot until you saw all the information.  I'll do a restart and see what happens.  In looking around myself, I am looking at any compatabiity issues using Tabs Outliner and The Great Suspender at the same time.  They have worked together for 6 months, so I doubt there is an issue.

**Comment by nompen** [comment]
I dislike testing multiple changes at once, but I needed tabs to work correctly today, so I disabled almost all other extensions, cleared all cache/custom settings, and restarted the computer.

All is well now.  I do not know what made the difference.  I have restored custom settings, and I will add back in all extensions over the next day or so.  If one of those seems to be causing a problem, I will start another issue and give you my findings.  

Thanks for your patience and assistance.
